packages/ts/react-crud/test/autogrid.spec.tsx

```diff
@@ -531,70 +531,76 @@ describe('@hilla/react-crud', () => {
         expect(grid.getBodyCellContent(1, 0)).to.have.rendered.text('Hello World 2');
       });
 
-      it('renders numbers as right aligned numbers', () => {
+      it('renders integers as right aligned numbers', () => {
         expect(grid.getBodyCellContent(0, 1)).to.have.style('text-align', 'end');
         expect(grid.getBodyCellContent(0, 1)).to.have.rendered.text('123,456');
         expect(grid.getBodyCellContent(1, 1)).to.have.rendered.text('-12');
       });
 
-      it('renders booleans as icons', () => {
-        expect(grid.getBodyCellContent(0, 2).querySelector('vaadin-icon')).to.have.attribute('icon', 'lumo:checkmark');
-        expect(grid.getBodyCellContent(1, 2).querySelector('vaadin-icon')).to.have.attribute('icon', 'lumo:minus');
+      it('renders decimals as right aligned numbers', () => {
```

Comment:
Could we have a couple of more tests what is happening in the following scenarios:

* formatting 123.784
* formatting 123.7 (would there be zero padding?)
* formatting 0.1 (would there be zero padding?)

Reply:
Sure, added two more cases for zero padding.